fsharp-backend/tests/httptestfiles/serve-latest.test

```diff
@@ -0,0 +1,50 @@
+[http-handler GET /]
+StaticAssets.serve_v1_ster "abcdef1234" "index.html"
+
+[canvas-name test-static-assets-deploy]
```

Comment:
Not a big deal, but I'd expect this at the top of the file for single-file tests